.github/workflows/fossa-scan.yml

```diff
@@ -0,0 +1,23 @@
+name: FOSSA Scans
+
+on:
+  workflow_call:
+
+jobs:
+    fossa-scan:
+        runs-on: ubuntu-latest
+        steps:
+          - name: "Checkout Code"
+            uses: actions/checkout@v3
+
+          - name: "Run FOSSA Scan"
+            uses: fossas/fossa-action@main
```

Comment:
is @main safe or do we wanna lock the version of it?

Reply:
let's lock if we can, version or commit id